packages/react-renderer-demo/src/docs-components/condition.md

```diff
@@ -25,7 +25,33 @@ You can show a field only if it meets a condition:
 }
 ```
 
-`when` - is name of field where the value is stored, **always required!**
+`when` - is name of field where the value is stored, **always required!**. It can be either string `'field-name'` or array of strings `['field-1', 'field-2']`. Conditions are using **OR** operator. If you want **AND** you can pass multiple conditions.
```

Comment:
>  Conditions are using **OR** operator. If you want **AND** you can pass multiple conditions.

This is really hard to understand :confused:

Reply:
I added explicit examples. I would be grateful for any suggestions.